jumpscale/sals/reservation_chatflow/reservation_chatflow.py

```diff
@@ -116,7 +129,13 @@ def update(self, tid, currency=None, bot=None):
                 j.sals.reservation_chatflow.wait_payment(
                     bot, rid, threebot_app=True, reservation_create_resp=reservation_create
                 )
-            return self._sal.wait_reservation(self._bot, rid)
+            wait_reservation_results = self._sal.wait_reservation(self._bot, rid)
+            # Update solution saved locally
+            explorer_name = self._sal._explorer.url.split(".")[1]
+            solution = self.solutions.get(f"{explorer_name}_{self.resv_id}")
```

Comment:
it feels like this would work only during the first update
as next time the self.rev_id will be the updated reservation id.

I would suggest deleting the old network solution and creating a new one with the correct name

when this is done you can also update `self.resv_id = rid`

Reply:
Yes it would only work once, apparently when I tested it, the updates were done already from the explorer so it wasn't reflected on multiple child networks.